search/process/tests.py

```diff
@@ -38,9 +38,9 @@ def test_authors(self):
             ]
         })
         doc = transform.to_search_document(meta)
-        self.assertEqual(doc.authors[0]['first_name'], 'B Ivan')
+        self.assertEqual(doc.authors[0]['first_name'], 'B. Ivan')
```

Comment:
Should we test both cases for search? Seems requiring periods is going to be another friction point for UI.